intel_nn_hal/GnaPreparedModel.cpp

```diff
@@ -224,6 +224,9 @@ bool GnaPreparedModel::initialize(const hidl_vec<hidl_handle>& modelCache, const
     gnaPluginPtr->loadNetwork(passed_network, isDecoderNw);
     time_point gnabuild_end = now();
     runtimeMetrics.nw_load_time = (double(millisecondsDuration(gnabuild_end, irbuild_end)));
+    for (auto item:mModelBlobs) {
+        item->deallocate();
+    }
```

Comment:
This change should also be part of build when perf counters are disabled..

Reply:
Thanks for catching that. Done . Please help review aain,